example/package.json

```diff
@@ -11,18 +11,18 @@
   "dependencies": {
     "@bringg/react-native-bringg-driver-sdk": "file:..",
     "@react-native-community/datetimepicker": "^2.4.0",
-    "react": "^16.13.1",
-    "react-native": "^0.63.4"
+    "react": "^17.0.1",
```

Comment:
used this helper tool:
https://react-native-community.github.io/upgrade-helper/?from=0.62.2&to=0.64.2